includes/compat/subscriptions/trait-wc-payments-subscriptions-utilities.php

```diff
@@ -120,13 +120,17 @@ public function get_subscriptions_plugin_version() {
 	}
 
 	/**
-	 * Gets the version of the Subscriptions base library running.
-	 *
-	 * This may be the version of the package in WC Payments or WC Subscriptions. Which ever one happens to be loaded.
+	 * Gets the version of the Subscriptions core library running.
 	 *
 	 * @return null|string The core Subscriptions libary version.
```

Comment:
```suggestion
	 * @return null|string The core Subscriptions library version.
```
Suggestion: fix typo in docblock.

Reply:
Or perhaps this could be `The Subscriptions core library version`.

Reply:
Maybe even `subscriptions-core` to be 100% consistent naming? Up to y'all.